src/screens/LocationPage/LocationPage.tsx

```diff
@@ -42,17 +39,7 @@ function LocationPage({ region }: LocationPageProps) {
           mobileMenuOpen={mobileMenuOpen}
           setMobileMenuOpen={setMobileMenuOpen}
         />
```

Comment:
now ChartsHolder is responsible for getting its own projections and loading them, so we can render it unconditionally.